src/kn10pag.c

```diff
@@ -1492,7 +1494,10 @@ ioinsdef(io_do_pi)
     register w10_t w;
     w = vm_read(e);		/* get lights data */
 
+#if KLH10_DEV_LITES
     lights_pgmlites(LHGET(w), RHGET(w));
+#endif
```

Comment:
YES. These additions are ok. (Although, looking carefully, also not needed, since the header file, if it is included, stubs out this call anyway).